### PR TITLE
Prevent spam order submitting

### PR DESCRIPTION
**Pull request by alexstotsky**
### PR description
- Prevented submitting next order before receiving notification about processing previous
- Cleaned outdated `change exchange` related code from `OrderForm`

### CODE REVIEW

**Comment by avsek477** [comment]
lgtm :+1:

**Review by avsek477 — APPROVED** [review]

File: src/components/OrderForm/OrderForm.container.js [hunk 1]
```diff
@@ -55,12 +56,15 @@ const mapDispatchToProps = dispatch => ({
       ...packet,
     }]))
   },
+
```
Comment (robertkowalski) [inline src/components/OrderForm/OrderForm.container.js:59]: ah, some air <3

File: src/redux/selectors/notifications/get-latest-notification-ts.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import _get from 'lodash/get'
+import { REDUCER_PATHS } from '../../config'
+
+const path = REDUCER_PATHS.NOTIFICATIONS
+
+export default (state) => {
+  return _get(state, `${path}.0.mts`, null)
```
Comment (robertkowalski) [inline src/redux/selectors/notifications/get-latest-notification-ts.js:7]: when you `console.log` this you will see that its set very fast, for every order book entry thats coming in...

so we can't reliably use it for determining if an order was submitted / sent

we should wait for an `on` message

**Review by robertkowalski — CHANGES_REQUESTED** [review]
the updates coming in (trades, orderbook) in a usual setup are too many to reliably use that as a confirmation if an order was sent

that needs some overhaul to make this a reliable feature, but the algo popups def have priority

**Comment by alexstotsky** [comment]
Ok then, will need to investigate which approach better be used here instead of the current based on notifications. What ways to handle it we currently have and what changes should be added to checking the order processing status in an optimal way. A different approach will be done in separate pr, this one will be closed.

**Pull request closed without merging**